node_modules/col-tests/lib/util.js

```diff
@@ -18,12 +18,14 @@ var config = require('config');
 var randomstring = require('randomstring');
 var util = require('util');
 
+var AssetsTestUtil = require('col-assets/tests/util');
+var LtiTestUtil = require('col-lti/tests/util');
```

Comment:
It's probably OK, but the time the exported functions get executed we should be in a next tick and each method should be available (I think)